### ISSUE TITLE
Update node-neuter to fix a crash ( Cannot read property 'type' of undefined)

### ISSUE THREAD

**Issue by Nivl** [squarewolf/gulp-neuter#9]
I have a crash when building the JS files.

```
if (childNode.type === 'FunctionDeclaration' ||
	         ^
TypeError: Cannot read property 'type' of undefined
```

I looked around and I realized that the bug was coming from the current version of node-neuter (0.1.2). By replacing the node-neuter files (included by gulp-neuter) by the one from the master branch, the bug goes away. So in order to make everything working, you need to use the code from the master branch of node-neuter.

see https://github.com/squarewolf/node-neuter/issues/5

**Comment by tvervest** [comment]
I've just pushed version 0.1.3 which uses 0.1.2 (as per PR #6). Could you verify that your issue is solved in the latest version?

**Comment by scottmessinger** [comment]
I have version 0.1.3 and I'm running into the same issue. Anything I can do to help?

**Comment by tvervest** [comment]
Sorry for the long wait, it was an issue in the node-neuter package. It should be resolved in 0.1.4
